src/main/java/org/dependencytrack/upgrade/v4120/v4120Updater.java

```diff
@@ -286,4 +287,18 @@ private void dropAuthorColumns(final Connection connection) throws SQLException
         }
     }
 
+
+
+    private void setInitialIsLatestFlags(Connection connection) throws SQLException {
+        LOGGER.info("Setting IS_LATEST flag to false for all Project entries");
+        try (final Statement stmt = connection.createStatement()) {
+            stmt.executeUpdate("""
+                    UPDATE "PROJECT" SET "IS_LATEST" = false
+                    """);
+            stmt.executeUpdate("""
+                    UPDATE "POLICY" SET "ONLY_FOR_LATEST_PROJECT_VERSION" = false
+                    """);
+        }
+    }
```

Comment:
Analog to my comment in the `Project` class - This seems to be redundant. DataNucleus will set the specified default value in the schema.